xastropy/xguis/igmguesses.py

```diff
@@ -158,29 +146,29 @@ def __init__(self, ispec, parent=None, previous_file=None,
         self.n_max_tuple = n_max_tuple
         self.min_strength = min_strength
 
-        # Spectrum
+        # Load spectrum
         spec, spec_fil = ltgu.read_spec(ispec)
-        # Normalize if needed
-        spec.normalize()
+        # Normalize
+        spec.normalize(co=spec.data[0]['co'])
```

Comment:
If the continuum is already loaded, you can
normalize just by setting normed, ie.

spec.normed = True

This is a better way.